server/service.go

```diff
@@ -129,6 +147,62 @@ func (m *BoostService) handleStatus(w http.ResponseWriter, req *http.Request) {
 	fmt.Fprintf(w, `{}`)
 }
 
+// sendValidatorPreferences is used to send the validators preferences to the registered relays
+func (m *BoostService) sendValidatorPreferences(log *logrus.Entry, payload []types.SignedValidatorRegistration) uint64 {
+	// We need a wait group to manage each routine used to perform the requests.
+	var wg sync.WaitGroup
+
+	// Use an atomic counter to count successful requests.
+	numSuccessRequestsToRelay := uint64(0)
+
+	// Send the validators preferences to each registered relay.
+	for _, relay := range m.relays {
+		wg.Add(1)
+
+		go func(relayAddr string) {
+			defer wg.Done()
+
+			url := relayAddr + pathRegisterValidator
+			log := log.WithField("url", url)
+
+			err := SendHTTPRequest(context.Background(), m.httpClient, http.MethodPost, url, payload, nil)
+			if err != nil {
+				log.WithError(err).Warn("error in registerValidator to relay")
+				return
+			}
+
+			atomic.AddUint64(&numSuccessRequestsToRelay, 1)
+		}(relay.Address)
+	}
+
+	wg.Wait()
+
+	return numSuccessRequestsToRelay
+}
+
+func (m *BoostService) registerValidatorAtInterval(interval time.Duration) {
+	var payload []types.SignedValidatorRegistration
+	log := m.log.WithField("method", "registerValidatorAtInterval")
+
+	ticker := time.NewTicker(interval)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-m.ctx.Done():
+			// mev-boost has probably stopped
+			return
+		case srr := <-m.registrationsRequests:
+			// registerValidator has received new registrations and forwards them to here
+			srr.numSuccessRequestsToRelay <- m.sendValidatorPreferences(log, srr.preferences)
```

Comment:
`payload` isn't updated

Reply:
👍 
Fixed in a3d5a4b578ce5c133f749112f71ba096230488f4